src/app/ApplicationTemplate.Tests/Framework/IntegrationTestBase.cs

```diff
@@ -117,6 +125,11 @@ protected virtual async Task ClearStorage()
 		}
 	}
 
+	/// <summary>
+	/// Gives access to the services and their configuration with a specific refence to the tested service.
+	/// Normally used to test Endpoints.
+	/// </summary>
+	/// <typeparam name="TSUT">The refence to the service under test.</typeparam>
```

Comment:
`TSUT` is a type parameter so it's definitely not a reference.
That comment would however apply to the `SUT` property.